website/common/locales/en/newClient.json

```diff
@@ -288,6 +288,11 @@
   "haveHatchablePet": "You have a <%= potion %> hatching potion and <%= egg %> egg to hatch this pet! <b>Click</b> the paw print to hatch.",
   "welcomeBack": "Welcome back!",
   "checkOffYesterDailies": "Check off any Dailies you did yesterday:",
+  "wizardText": "Mages learn swiftly, gaining Experience and Levels faster than other classes. They also get a great deal of Mana for using special abilities. Play a Mage if you enjoy the tactical game aspects of Habitica, or if you are strongly motivated by leveling up and unlocking advanced features!",
+  "chooseClass": "Choose your Class",
+  "chooseClassLearnMarkdown": "[Learn more about Habitica's class system](http://habitica.wikia.com/wiki/Class_System)",
+  "selectClass": "Select <%= heroClass %>",
+  "wizard": "Mage",
```

Comment:
I'm fine with renaming the strings to wizard but we should make sure it's used everywhere in the new client. In particular for the string below that is quite long, so that we don't have to remember to update it in two places